internal/exec/helmfile.go

```diff
@@ -50,17 +50,19 @@ func ExecuteHelmfile(cmd *cobra.Command, args []string) error {
 	}
 
 	// Write variables to a file
-	varFile := fmt.Sprintf("%s-%s-%s.helmfile.vars.yaml", info.ContextPrefix, info.ComponentFolderPrefix, info.Component)
+	var varFile string
 	var varFileName string
 
 	if len(info.ComponentFolderPrefix) == 0 {
+		varFile = fmt.Sprintf("%s-%s.helmfile.vars.yaml", info.ContextPrefix, info.Component)
 		varFileName = path.Join(
 			c.Config.BasePath,
 			c.Config.Components.Helmfile.BasePath,
 			info.Component,
 			varFile,
 		)
 	} else {
+		varFile = fmt.Sprintf("%s-%s-%s.helmfile.vars.yaml", info.ContextPrefix, info.ComponentFolderPrefix, info.Component)
```

Comment:
Why not use strings.join instead ?

Reply:
it operates on slices (lists), so would have to create a list first, which is unnecessary